beam-core/Database/Beam/Backend/SQL/Row.hs

```diff
@@ -25,6 +25,8 @@ import           Data.Typeable
 import           Data.Vector.Sized (Vector)
 import qualified Data.Vector.Sized as Vector
 
+import qualified Control.Monad.Fail as Fail
```

Comment:
Why is this imported qualified?

Reply:
Because on GHC-8.6 (and older) `fail` and `Fail.fail` are different